src/components/containers/SingleArticle.jsx

```diff
@@ -130,7 +155,12 @@ class SingleArticle extends Component {
                 </div>
                 {isVerified && viewingUser !== author.username && (
                   <div className="follow-bookmark-box">
-                    <button className="follow-btn">Follow</button>
+                    <button
+                    className="follow-btn"
+                    onClick={this.UserFollow.bind(this, articleAuthor, buttonAction)}
```

Comment:
Do you still need to bind here even after declaring the function as an arrow function?